change/@fluentui-react-components-059500c6-1617-4bd9-9f42-ea4726c5ece3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
This should probably be none (and dependentChangeType none) since this doesn't really affect the published package, at least how we've been thinking about it for vnext